File: UI/src/app/cluster.ts

import { ClusterJobDocument } from "./ClusterJobDocument"; 
import { ClusterJobTerm } from "./ClusterJobTerm"; 

export interface Cluster 
{ 
    id: number;
    jobId: number;
    clusterCalculationId: number;
    centerVector: number[];
    si?: number;
    clusterJobDocuments: ClusterJobDocument[];
    clusterJobTerms: ClusterJobTerm[];
}

File: UI/ViewModels/Dtos/Cluster.cs
using System.Collections.Generic;

namespace UI.ViewModels.Dtos
{
    public class Cluster
    { 
        public int Id { get; set; }
        public int JobId { get; set; }
        public int ClusterCalculationId { get; set; }
        public float[] CenterVector { get; set; }
        public float? Si { get; set; }

        public IEnumerable<ClusterJobDocument> ClusterJobDocuments { get; set; }
        public IEnumerable<ClusterJobTerm> ClusterJobTerms { get; set; }
    }
}